src/main/haskell/matching-logic/test/Paths.hs

```diff
@@ -0,0 +1,20 @@
+{-# LANGUAGE TemplateHaskell #-}
+
+module Paths where
+
+import qualified Language.Haskell.TH     as TH
+import qualified Language.Haskell.TH.Lib as TH
+import           System.FilePath         (takeDirectory, (</>))
+
+
+-- | Resolve the relative file name of a test resource.
```

Comment:
I think this should interpret the paths relative to `src/main/haskell/matching-logic`, which is where the existing paths have been relative to. `pwd </> ".." </> relative` should do it.

Reply:
A more general way to do this would be traverse up the directory tree until we find `package.yaml`. That would eliminate the need to change this file if we move the module, too.